etc/uid.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
```

Comment:
Python script for pulling the UID regex out of cylc-flow and dropping it in here.

Could run in a GH action if being serious, not done this yet.

Reply:
Might be useful to put this description in a docstring for this script